src/commands/moderation/prefixCommands/functions/addCommand.ts

```diff
@@ -0,0 +1,160 @@
+    color: Colors.Red,
+});
+
+const wrongFormatEmbed = (invalidString: string) => makeEmbed({
+    title: 'Prefix Commands - Add Command - Wrong format',
+    description: `The name and aliases of a command can only contain alphanumerical characters, underscores and dashes. ${invalidString} is invalid.`,
+    color: Colors.Red,
+});
+
+const categoryNotFoundEmbed = (category: string) => makeEmbed({
+    title: 'Prefix Commands - Add Command - Category not found',
+    description: `The prefix command category ${category} does not exist. Please create it first.`,
+    color: Colors.Red,
+});
+
+const alreadyExistsEmbed = (command: string) => makeEmbed({
+    title: 'Prefix Commands - Add Command - Already exists',
+    description: `The prefix command ${command} already exists. Not adding again.`,
+    color: Colors.Red,
+});
+
+const successEmbed = (command: string) => makeEmbed({
+    title: `Prefix command ${command} was added successfully.`,
+    color: Colors.Green,
+});
+
+const modLogEmbed = (moderator: User, command: string, aliases: string[], description: string, isEmbed: boolean, embedColor: string, commandId: string) => makeEmbed({
+    title: 'Prefix command added',
+    fields: [
+        {
+            name: 'Command',
+            value: command,
+        },
+        {
+            name: 'Moderator',
+            value: `${moderator}`,
+        },
+        {
+            name: 'Aliases',
+            value: aliases.join(','),
+        },
+        {
+            name: 'Description',
+            value: description,
+        },
+        {
+            name: 'Is Embed',
+            value: isEmbed ? 'Yes' : 'No',
+        },
+        {
+            name: 'Embed Color',
+            value: embedColor || '',
+        },
+    ],
+    footer: { text: `Command ID: ${commandId}` },
+    color: Colors.Green,
+});
+
+const noModLogs = makeEmbed({
+    title: 'Prefix Commands - Add Command - No Mod Log',
+    description: 'I can\'t find the mod logs channel. Please check the channel still exists.',
+    color: Colors.Red,
+});
+
+export async function handleAddPrefixCommand(interaction: ChatInputCommandInteraction<'cached'>) {
+    await interaction.deferReply({ ephemeral: true });
+
+    const conn = getConn();
+
+    if (!conn) {
+        await interaction.followUp({ embeds: [noConnEmbed], ephemeral: true });
+        return;
+    }
+
+    const name = interaction.options.getString('name')!;
+    const category = interaction.options.getString('category')!;
+    const description = interaction.options.getString('description')!;
+    const aliasesString = interaction.options.getString('aliases') || '';
+    const aliases = aliasesString !== '' ? aliasesString.split(',') : [];
+    const isEmbed = interaction.options.getBoolean('is_embed') || false;
+    const embedColor = interaction.options.getString('embed_color') || '';
+    const moderator = interaction.user;
+
+    const nameRegex = /^[\w\d-_]+$/;
```

Comment:
```suggestion
    const nameRegex = /^[\w-_]+$/;
```

Is the \d redundant with the \w